libs/server/Storage/Session/ObjectStore/SortedSetOps.cs

```diff
@@ -927,5 +927,58 @@ private GarnetStatus SortedSetDifference<TObjectContext>(ReadOnlySpan<ArgSlice>
 
             return GarnetStatus.OK;
         }
+
+        /// <summary>
+        /// Removes and returns up to count members and their scores from the first sorted set that contains a member.
+        /// </summary>
+        public unsafe GarnetStatus SortedSetMPop(ReadOnlySpan<ArgSlice> keys, int count, bool lowScoresFirst, out ArgSlice poppedKey, out (ArgSlice member, ArgSlice score)[] pairs)
+        {
+            pairs = default;
+            poppedKey = default;
+
+            if (keys.Length == 0)
+                return GarnetStatus.OK;
+
+            var createTransaction = false;
```

Comment:
I am not sure If I need to use transaction for this comment as the other similar command in Garnet doesn't have transaction

Reply:
It depends if this operation should be atomic or not, I don't see any promise of atomicity in the spec. We don't treat LMPOP as a transaction either, I think that's probably on purpose.

Reply:
What should I do here? Personally I think we need transaction

Reply:
Please implement it as a txn.

Reply:
Then we are good right, I already have transaction as part of this PR